packages/fxa-content-server/app/scripts/lib/router.js

```diff
@@ -228,7 +228,16 @@ Router = Router.extend({
         }),
       });
     },
-    'force_auth(/)': createViewHandler(ForceAuthView),
+    'force_auth(/)': function () {
+      this.createReactOrBackboneViewHandler('force_auth', ForceAuthView, {
+        // see comment in fxa-settings/src/pages/Signup/container.tsx for param explanation
+        email: this.user.get('emailFromIndex'),
+        ...(this.user.get('emailFromIndex') && {
+          emailStatusChecked: 'true',
```

Comment:
Can users reach `force_auth` after the index page?

If it's just through Sync, we should be able to remove all of these params and just include `...Url.searchParams(this.window.location.search),`. If you include `emailStatusChecked: 'true',` then `accountStatusByEmail` or whatever it is won't get called.